doc/classes/MultiMesh.xml

```diff
@@ -87,28 +90,33 @@
 		<member name="buffer" type="PackedFloat32Array" setter="set_buffer" getter="get_buffer" default="PackedFloat32Array()">
 		</member>
 		<member name="color_array" type="PackedColorArray" setter="_set_color_array" getter="_get_color_array">
+			See [method set_instance_color].
 		</member>
 		<member name="custom_data_array" type="PackedColorArray" setter="_set_custom_data_array" getter="_get_custom_data_array">
+			See [method set_instance_custom_data].
 		</member>
 		<member name="instance_count" type="int" setter="set_instance_count" getter="get_instance_count" default="0">
 			Number of instances that will get drawn. This clears and (re)sizes the buffers. Setting data format or flags afterwards will have no effect.
 			By default, all instances are drawn but you can limit this with [member visible_instance_count].
 		</member>
 		<member name="mesh" type="Mesh" setter="set_mesh" getter="get_mesh">
-			Mesh to be drawn.
+			[Mesh] resource to be instanced.
+			The looks of the individual instances can be modified using [method set_instance_color] and [method set_instance_custom_data].
 		</member>
 		<member name="transform_2d_array" type="PackedVector2Array" setter="_set_transform_2d_array" getter="_get_transform_2d_array">
+			See [method set_instance_transform_2d].
 		</member>
 		<member name="transform_array" type="PackedVector3Array" setter="_set_transform_array" getter="_get_transform_array">
+			See [method set_instance_transform].
 		</member>
 		<member name="transform_format" type="int" setter="set_transform_format" getter="get_transform_format" enum="MultiMesh.TransformFormat" default="0">
 			Format of transform used to transform mesh, either 2D or 3D.
 		</member>
 		<member name="use_colors" type="bool" setter="set_use_colors" getter="is_using_colors" default="false">
-			If [code]true[/code], the [MultiMesh] will use color data (see [member color_array]).
+			If [code]true[/code], the [MultiMesh] will use color data (see [method set_instance_color]).
```

Comment:
```suggestion
			If [code]true[/code], the [MultiMesh] will use color data (see [method set_instance_color]). Can only be set when [member instance_count] is ``0`` or less.
```

Reply:
Done, plus added conclusion `This means that you need to call this method before calling [method set_instance_count], or temporarily reset your instance count to ``0``.` to avoid (even brief) confusion.